### DISCUSSION TITLE
致谢落款如何实现？

### DISCUSSION THREAD

**Discussion by CandyTt20** [nju-lug/NJUThesis#120] [Q&A] [answered]
<img width="1015" alt="Screen Shot 2022-04-04 at 11 37 49" src="https://user-images.githubusercontent.com/63237169/161470228-ee8e7973-72ba-4482-92e2-1b1319ae38b6.png">
如图，格式包括：位置出现在右下角，姓名的字体为楷体。

**Comment by atxy-blip** [comment, marked as answer]
```TeX
\begin{flushright}
\begin{minipage}{10em}
\centering
{\kaishu \large 某某}

某年某月某日

南京大学物理楼304
\end{minipage}
\end{flushright}
```

**Reply by CandyTt20** [reply]
多谢🙏
